consts/consts.go

```diff
@@ -53,15 +53,18 @@ const (
 	CT_LIKUTIM               = "LIKUTIM"
 
 	// Content types for additional Elastic results
-	SCT_BLOG_POST = "R_BLOG_POST"
-	SCT_TWEET     = "R_TWEET"
+	SCT_TWEET = "R_TWEET"
 
 	// Content Role types
 	CR_LECTURER = "LECTURER"
 
 	// Persons patterns
 	P_RAV = "rav"
 
+	// Persons UIDs
```

Comment:
I think you can use [this](https://github.com/Bnei-Baruch/archive-backend/blob/9b798744d583894fda1a2439468cb2ea42c9dd5e/mdb/registry.go#L88)

Reply:
Done, so we don't have hard coded UIDs but the code is not that preaty....

mdb.PERSONS_REGISTRY.ByPattern[consts.P_RAV].UID
mdb.PERSONS_REGISTRY.ByPattern[consts.P_RABASH].UID